src/git/runner.go

```diff
@@ -682,6 +683,84 @@ func (r *Runner) LocalAndRemoteBranches() ([]string, error) {
 	return MainFirst(result), nil
 }
 
+// LocalBranches returns the names of all branches in the local repository,
+// ordered alphabetically.
+func (r *Runner) LocalBranches() (result []string, err error) {
+	res, err := r.Run("git", "branch")
+	if err != nil {
+		return result, err
+	}
+	for _, line := range res.OutputLines() {
+		line = strings.Trim(line, "* ")
+		line = strings.TrimSpace(line)
+		result = append(result, line)
+	}
+	return
+}
+
+// LocalBranchesMainFirst provides the names of all local branches in this repo.
+func (r *Runner) LocalBranchesMainFirst() (result []string, err error) {
```

Comment:
I'm confused about the difference between this and `LocalBranchesWithMainBranchFirst`

Reply:
Rightfully so, they are the same! Thanks for catching this.

Reply:
You are - as always - refreshingly meticulous!